404.html

```diff
@@ -15,12 +15,12 @@
 <!--Fallback page, don't allow indexing-->
 <meta name="robots" content="noindex">
 <h1>An open source P2P digital currency</h1>
+{% comment %}
 <p style="font-size:125%;">Bitcoin is a currency, a protocol, and a software that enables
 <ul style="font-size:125%">
-  <li>Instant peer to peer transactions</li>
```

Comment:
I should note that this is the 404 page and that it redirects to the home page for anyone with Javascript enabled, so we probably don't need to spend much time optimizing it.  (Frankly, I think it should say "page not found" instead of being an old version of the index page.)